packages/next/build/index.ts

```diff
@@ -882,6 +887,7 @@ export default async function build(
           files: [
             ROUTES_MANIFEST,
             path.relative(distDir, manifestPath),
+            path.relative(distDir, appManifestPath),
```

Comment:
This should only be added if `config.experimental.appDir` is true as it's not generated when `false`.

Reply:
I've added the condition :)